### PR TITLE
[DOXIA-717] Update org.codehaus.plexus:plexus-testing to 1.3.0

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -188,8 +188,8 @@ under the License.
       <dependency>
         <groupId>com.google.inject</groupId>
         <artifactId>guice</artifactId>
-        <version>4.2.3</version>
-        <classifier>no_aop</classifier>
+        <version>6.0.0</version>
+        <scope>test</scope>
```
Comment (michael-o) [inline pom.xml:192]: How does this relate to Plexus Testing?

Comment (slachiewicz) [reply pom.xml:192]: Plexus testing uses it. With the old version -  tests will fail

Comment (michael-o) [reply pom.xml:192]: Alright, I see. Please leave a comment in the commit message so that this information does not get lost.

**Review by michael-o — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #968

**Pull request merged**